packages/commonwealth/client/scripts/views/pages/CommunityManagement/AdminsAndModerators/AdminsAndModerators.tsx

```diff
@@ -25,18 +23,17 @@ const AdminsAndModerators = () => {
 
   const { data: { admins: returnedAdmins, mods: returnedMods } = {} } =
     useFetchAdminQuery({
-      chainId: app.activeChainId(),
+      communityId: app.activeChainId(),
     });
 
   const { data: searchResults, refetch } = useSearchProfilesQuery({
     communityId: app.activeChainId(),
     searchTerm: debouncedSearchTerm,
-    limit: 20,
+    limit: 4000,
```

Comment:
Api limits it to `100` i think, for now we can also use `100`. I am fine with `4000` as well, but it will be confusing for anyone developing on this further, i suggest we use `100` here

Reply:
bump on this @Israellund

Reply:
bump on this @Israellund